### ISSUE TITLE
Zen Kernel Uses PDS by Default with No Obvious Way to Switch to CFS.

### ISSUE THREAD

**Issue by EmadGhazal** [zen-kernel/zen-kernel#301]
According to [heftig](https://github.com/zen-kernel/zen-kernel/issues/165#issuecomment-586843201), the Zen Kernel uses the CFS scheduler. When building it with default config, however, it turns out to be using the PDS scheduler, with no option to change the scheduler through menuconfig.

Am I doing something or is my understanding wrong? Or is the quoted information outdated and the Zen Kernel no longer uses the CFS scheduler?

Here's the output of `uname -rs`:

> Linux 6.2.2-zen1

And here's the dmesg output indicating the schduler used:

> [    0.036031] sched/pds: PDS CPU Scheduler v6.2-r0 by Alfred Chen.

**Comment by heftig** [comment]
We should be using the CFS scheduler.

`/proc/config.gz` contains `# CONFIG_SCHED_ALT is not set` and I do not have the reported message in dmesg.

**Comment by damentz** [comment]
The official config disables CONFIG_SCHED_ALT [1], so it is a custom build.  The solution is to turn it off in your config for your customized kernel.  Closing issue since this doesn't affect the official build.

[1] https://github.com/archlinux/svntogit-packages/blob/packages/linux-zen/trunk/config#L203

```
#
# Scheduler features
#
CONFIG_UCLAMP_TASK=y
CONFIG_UCLAMP_BUCKETS_COUNT=5
# CONFIG_SCHED_ALT is not set
# end of Scheduler features
```

**Comment by EmadGhazal** [comment]
Sorry for not making it clear in my report. Yes. I'm building the Zen Kernel myself. No binaries are available for Debian.

I cloned the repo again, ran `make menuconfig`, didn't change a thing, saved the .config file, inspected it, and indeed, `# CONFIG_SCHED_ALT is not set` was commented out. Built the kernel again, the dmesg output regarding PDS disappeared. This is so weird because I did exactly the same thing in my first build, but this indeed solves the problem.

Sadly, the Zen Kernel with CFS doesn't improve the system responsiveness under heavy memory load. The system locks up just as with the stock kernel and my only option then is to manually trigger the OOM killer or hard-reboot the system. Too sad that PDS doesn't support big.LITTLE, because literally no other scheduler comes even close to its performance under heavy memory load.

Appreciate your support and thank you for your tremendous work on Zen and Liquorix.
